db/migrate/20240410165735_create_pega_table.rb

```diff
@@ -0,0 +1,19 @@
+class CreatePegaTable < ActiveRecord::Migration[7.1]
+  def change
+    create_table :pega_tables do |t|
+      t.uuid :uuid
+      t.string :veteranfirstname
+      t.string :veteranmiddlename
+      t.string :veteranlastname
+      t.string :sponsorfirstname
+      t.string :sponsormiddlename
+      t.string :sponsorlastname
```

Comment:
For 10-10d the sponsor and the Veteran are the same person so this naming is somewhat unclear - will these columns have the same data? Just checking, thanks!

Reply:
Oh that's right! I got mixed up.